packages/trpc/server/middlewares/captureErrorsMiddleware.ts

```diff
@@ -5,16 +5,15 @@ import { redactError } from "@calcom/lib/redactError";
 import { middleware } from "../trpc";
 
 const captureErrorsMiddleware = middleware(async ({ next }) => {
-  const result = await next();
-  if (result && !result.ok) {
-    const cause = result.error.cause;
-    if (!cause) {
-      return result;
-    }
-    captureException(cause);
-    throw redactError(cause);
+  try {
+    const result = await next();
+    if (result.ok) return result;
+    if (!result.error.cause) return result;
+    throw result.error.cause;
```

Comment:
I do agree with mrge on this one that this is redundant in a try/catch block, I'll still approve since it does the job

Reply:
https://github.com/calcom/cal.com/pull/20718#discussion_r2045524537